src/app/nation/nation.component.ts

```diff
@@ -146,16 +136,16 @@ export class NationComponent implements OnInit, AfterViewInit {
     const validation = {
       adminName: [ '', Validators.required,
         // an arrow function is for lexically binding 'this' otherwise 'this' would be undefined
-        ac => this.nationValidatorService.nationCheckerService$(ac)
       ],
-      name: [ '', Validators.required ],
-      nationUrl: [ '', Validators.required ],
+      name: [ '', Validators.required, ac => this.nationValidatorService.checkNationExists$(ac) ],
```

Comment:
it was supposed to use mango queries selector. but it is still using single field query